qiime/split_libraries_lea_seq.py

```diff
@@ -103,7 +107,8 @@ def get_LEA_seq_consensus_seqs(fwd_read_f, rev_read_f,
                                min_reads_per_random_bc,
                                min_difference_clusters,
                                barcode_column,
-                               reverse_primer_column):
+                               reverse_primer_column,
+                               phred_offset):
```

Comment:
This script isn't using the quality scores that are parsed from the fastq files, so I don't think you need a phred offset option.